class.jetpack.php

```diff
@@ -3548,97 +3548,6 @@ function plugin_action_links( $actions ) {
-				<div class="jp-banner__description-container">
-					<h2 class="jp-banner__header"><?php esc_html_e( 'Jetpack Centralized Site Management', 'jetpack' ); ?></h2>
-					<p class="jp-banner__description"><?php printf( __( 'Manage multiple Jetpack enabled sites from one single dashboard at wordpress.com. Allows all existing, connected Administrators to modify your site.', 'jetpack' ), 'https://jetpack.com/support/site-management' ); ?></p>
-					<p class="jp-banner__button-container">
-						<a href="<?php echo esc_url( $opt_in_url ); ?>" class="button button-primary" id="wpcom-connect"><?php _e( 'Activate Jetpack Manage', 'jetpack' ); ?></a>
-						<a href="https://jetpack.com/support/site-management" class="button" target="_blank" title="<?php esc_attr_e( 'Learn more about Jetpack Manage on Jetpack.com', 'jetpack' ); ?>"><?php _e( 'Learn more', 'jetpack' ); ?></a>
-					</p>
-				</div>
-		</div>
-		<?php
-	}
-
-	/**
-	 * Returns the url that the user clicks to remove the notice for the big banner
-	 * @return (string)
-	 */
-	function opt_out_jetpack_manage_url() {
-		$referer = '&_wp_http_referer=' . add_query_arg( '_wp_http_referer', null );
-		return wp_nonce_url( Jetpack::admin_url( 'jetpack-notice=jetpack-manage-opt-out' . $referer ), 'jetpack_manage_banner_opt_out' );
-	}
-	/**
-	 * Returns the url that the user clicks to opt in to Jetpack Manage
-	 * @return (string)
-	 */
-	function opt_in_jetpack_manage_url() {
-		return wp_nonce_url( Jetpack::admin_url( 'jetpack-notice=jetpack-manage-opt-in' ), 'jetpack_manage_banner_opt_in' );
-	}
-
-	function opt_in_jetpack_manage_notice() {
-		?>
-		<div class="wrap">
-			<div id="message" class="jetpack-message is-opt-in">
-				<?php echo sprintf( __( '<p><a href="%1$s" title="Opt in to WordPress.com Site Management" >Activate Site Management</a> to manage multiple sites from our centralized dashboard at wordpress.com/sites. <a href="%2$s" target="_blank">Learn more</a>.</p><a href="%1$s" class="jp-button">Activate Now</a>', 'jetpack' ), $this->opt_in_jetpack_manage_url(), 'https://jetpack.com/support/site-management' ); ?>
-			</div>
-		</div>
-		<?php
-
-	}
-	/**
-	 * Determines whether to show the notice of not true = display notice
-	 * @return (bool)
-	 */
-	function can_display_jetpack_manage_notice() {
-		// never display the notice to users that can't do anything about it anyways
-		if( ! current_user_can( 'jetpack_manage_modules' ) )
-			return false;
-
-		// don't display if we are in development more
-		if( Jetpack::is_development_mode() ) {
-			return false;
-		}
-		// don't display if the site is private
-		if(  ! Jetpack_Options::get_option( 'public' ) )
-			return false;
-
-		/**
-		 * Should the Jetpack Remote Site Management notice be displayed.
-		 *
-		 * @since 3.3.0
-		 *
-		 * @param bool ! self::is_module_active( 'manage' ) Is the Manage module inactive.
-		 */
-		return apply_filters( 'can_display_jetpack_manage_notice', ! self::is_module_active( 'manage' ) );
```

Comment:
added to the list.  This filter will now throw a deprecated notice for those using it